spec/lib/tenejo/csv_importer_spec.rb

```diff
@@ -83,10 +83,11 @@
       end
       let(:pf_collection) { Tenejo::PFCollection.new({ identifier: 'TEST0001', title: 'Importer test collection' }, -1) }
 
-      it "creates a new collection", :aggregate_failures do
+      it "creates a new collection" do
```

Comment:
Why don't you like :aggregate failures?  I like seeing all the failures at once when we have multiple expectations on slow test.